src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -12,6 +14,49 @@ class AuthApplicationSection extends React.Component {
     let content;
     const verifiedAccountType = 3;// verified ID.me accounts are type 3
     const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
+    const savedForms = this.props.userProfile.savedForms;
+    const hasSavedForms = !!savedForms;
+    
+    //TODO:  (ceh) verify edu and burial form codes
+    const formTitles = {
```

Comment:
I'd like to use the data from the `formConfig`s themselves, but this might be tricky since they don't contain the `path`.

Either way, that still doesn't solve the problem of making sure this file contains all SiP-enabled forms. I think @jbalboni's idea of throwing an error up to Sentry is great as a just-in-case measure, but I'd like to prevent that from happening.

Now this might be overkill, but bear with me. `vets-json-schema/dist/schemas` has a list of all the schemas, so if we use the keys there as a master list, we can then use them in a unit test that checks to see if the form is SiP-enabled, and if so, that this list (`formTitles`) contains the key (of the form ID).

```js
import formConfig as formConfig1990 from '../path/to/config';
import formConfig as formConfig1995 from '../path/to/config';
import sipEnabledForms from '../path/to/helper/file';
import schemas from 'vets-json-schema/dist/schemas';

const allFormIds = Object.keys(schemas);
const configsArray = [formConfig1990, formConfig1995];

const configs = {};

// Transmogrify the array of configs into an object
configsArray.forEach((formConfig) => {
  configs[formConfig.formId] = formConfig;
});

allFormIds.forEach((formId) => {
  describe(`${formId}`, () => {
    // To make sure we're at least checking to see if it needs to be in the
    //  list of SiP-enabled forms
    it('should be in the configsArray', () => {
      expect(configs[formId]).to.not.be.undefined;
    });

    if (configs[formId] && !configs[formId].disableSave) {
      it('should be in the list of SiP-enabled forms', () => {
        expect(sipEnabledForms[formId]).to.not.be.undefined;
      });
    }
  });
});
```
When :point_up: fails, we know we need to either add a new form to the SiP-enabled list, disable saving on that form, or add it to this unit test to perform the necessary checks on it.

What say ye?